### ISSUE TITLE
Support transfer learning

### ISSUE THREAD

**Issue by uschmidt83** [stardist/stardist#126]

**Comment by jjhbw** [comment]
Thanks for the excellent model and well curated codebase! StarDist turned out to be way easier to use than other cell segmentation models.

I was wondering, is there already a way for me to set up transfer learning? I'm perfectly willing to hack it together myself, if it doesn't require too much domain knowledge. My lab is primarily interested in cell classification on HE images.

**Comment by uschmidt83** [comment]
Yes, it can be done. Please see [this recent forum thread](https://forum.image.sc/t/how-to-continue-training/73482) on the topic.
